bin/templates/platform_www/cdv-electron-main.js

```diff
@@ -62,10 +89,27 @@ function createWindow () {
     });
 }
 
+function configureProtocol () {
+    protocol.registerFileProtocol(scheme, (request, _cb) => {
```

Comment:
Any reason for prefixing `cb`? Normally, I'll do this only if it's unused, but it's called.

Reply:
I prefix it to avoid the lint error

> error  Unexpected literal in error position of callback  `standard/no-callback-literal`

I could maybe disabled `no-callback-literal` with

```
// eslint-disable-line no-callback-literal
```

Reply:
I updated the `.eslintrc` to exclude all `standard/no-callback-literal` checks to that file.